src/components/TypesOfRecipeCards/CommentCard.jsx

```diff
@@ -0,0 +1,83 @@
+import Pizza from "../../assets/pizza.jpg";
+import User1 from "../../assets/user/User2.svg";
+
+const CommentCard = ({ user, review }) => {
+  //CommentCard grabs all the reviews created by the user
+  // user is the reviewer
+
+  const reviewContent = review ? review : null;
+
+  const orginalRecipe = reviewContent?.recipeId;
```

Comment:
What is original supposed to mean?

Reply:
I was trying to clarify the which recipe the user is reviewing